yield-generator-basic.py

```diff
@@ -156,9 +161,51 @@ def on_tx_confirmed(self, cjorder, confirmations, txid):
 
 
 def main():
-    load_program_config()
+    global txfee, cjfee_a, cjfee_r, ordertype, nickserv_password, minsize, mix_levels
```

Comment:
Either that or `self` as a parameter. I just wanted to explicitly specify which global variables are modified.